luci/luci-app-ffwizard-falter/po/en/ffwizard-falter.po

```diff
@@ -9,18 +9,47 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: en\n"
-"X-Generator: Poedit 2.4.2\n"
+"X-Generator: Poedit 3.0\n"
 
-#: applications/luci-app-ffwizard-falter/root/usr/share/rpcd/acl.d/ffwizard-berlin.json:3
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/assistent/wireless.lua:42
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:25
+msgid "2.4 Ghz Wifi ("
+msgstr "2.4 Ghz Wifi"
+
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/assistent/wireless.lua:44
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:27
+msgid "5 Ghz Wifi ("
+msgstr "5 Ghz Wifi"
+
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/assistent/wireless.lua:64
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:44
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:50
+msgid "802.11s"
+msgstr ""
+
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:45
+msgid ""
+"</strong>. The current default setup is 802.11s. Please select how to use "
+"this device in the future."
+msgstr ""
+
+#: luci/luci-app-ffwizard-falter/root/usr/share/rpcd/acl.d/ffwizard-berlin.json:3
 msgid "Access some status information without login"
 msgstr "Access some status information without login"
 
-#: applications/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:53
-#: applications/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/wireless-mesh.lua:54
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:44
+msgid "Ad-Hoc"
+msgstr ""
+
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/upgrades/005-wireless-mesh.lua:54
 msgid "Ad-Hoc (outdated)"
 msgstr "Ad-Hoc (outdated)"
 
-#: applications/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:10
+#: luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/assistent/wireless.lua:68
+msgid "Ad-Hoc (veraltet)"
+msgstr ""
```

Comment:
Missing translation?

Reply:
Yes, apparently that one was missing. For new apps, we should use english only. its quite confusing to translate that mixture from german and english. :D

Reply:
Issue #246 has been created to address this